.changeset/olive-avocados-attack.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/design-system-components": major
+---
+
+Allow `Checkmark`, `Checkbox` and `Radio` as `Hds::Dropdown::ListItem`s
+Rename `Hds::Dropdown::ListItem` internal CSS classes
```

Comment:
do you think it would make sense to add here a note/comment that explains that _this_ is the breaking change that causes the version bump to be major?

Reply:
[Updated the changelog entry](https://github.com/hashicorp/design-system/pull/1185/commits/fcf80f8b846bf615aa648335793e9a7d39d78c60) to list the actual changes and the potential effects.